### PR TITLE
Pre-release docs review

### PR DESCRIPTION
**Pull request by jayniep**

### CODE REVIEW

File: docs/source/configuration.md [hunk 1]
```diff
@@ -439,78 +433,77 @@ Parameters used to smooth the shape of the return paths that are displaced in th
...
-If `skip_sweep` is False and `save_stl_flag` is True, the generated result is saved in the output_directory, with a name corresponding 
-to `{project_name}_surface_part{part_ind}_{field_shape_function}.stl`, where `part_ind` is the zero-based index of the winding coil mesh parts.
+  If `skip_sweep` is False and `save_stl_flag` is True, the generated result is saved in the output_directory, with a name corresponding 
+  to `{project_name}_surface_part{part_ind}_{field_shape_function}.stl`, where `part_ind` is the zero-based index of the winding coil mesh parts.
 
-The `field_shape_function` is stripped of any `*`, `^`, and `,` symbols.
+  The `field_shape_function` is stripped of any `*`, `^`, and `,` symbols.
 
 
 ## Evaluate Results
```
Comment (jayniep) [inline docs/source/configuration.md:479]: Add a one sentence intro to what this section contains - as the other sections have.

File: docs/source/quick_start.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 While the **pyCoilGen** application can be used as a command-line application, due to the [large number of parameters](./configuration.md) to be configured, it is best used as part of a Python script. 
 
-This allows users to use the application, or previously saved data, to manipulate and/or view coil design parameters, field errors, etc.
```
Comment (jayniep) [inline docs/source/quick_start.md:5]: It is clearer to refer to `pyCoilGen`.

File: docs/source/quick_start.md [hunk 3]
```diff
@@ -1,13 +1,33 @@
...
 
-The following code snippet shows an example of a basic Python script that creates a dictionary of parameters that is then passed to the `pyCoilGen` function to determine the solution.
+`pyCoilGen` has a command line interface and a Python API.
+
+### Command Line Interface
+
+To call `pyCoilGen` from the command line: 
+
+```
+
```
Comment (jayniep) [inline docs/source/quick_start.md:14]: Make this a "template" command: `pycoilgen --parameter1 value1 --parameter2 value2 ....`

File: docs/source/quick_start.md [hunk 4]
```diff
@@ -1,13 +1,33 @@
...
+```
+
+For example:
+
+
+```
+
```
Comment (jayniep) [inline docs/source/quick_start.md:21]: Give an example here with specific parameters.

File: docs/source/quick_start.md [hunk 5]
```diff
@@ -1,13 +1,33 @@
...
+```
+
+You can access the help with `-h`. All parameters are also described in detail in the [Configuration](./configuration.md) chapter. 
 
+### Python API
```
Comment (jayniep) [inline docs/source/quick_start.md:26]: Would a user only need the `pyCoilGen` function?

File: docs/source/results.md [hunk 6]
```diff
@@ -1,36 +1,35 @@
...
-For example:
+For example, for a `project_name` of `ygradient_cylinder`, the `.stl` files are:
 
  - `ygradient_cylinder_surface_part0_y.stl` : Shows the winding coil surface for the first coil part.
  - `ygradient_cylinder_swept_layout_part0_y.stl` : Shows the 3D swept mesh of the winding coil conductor for the first coil part.
 
 ## CoilSolution
 
-The `CoilSolution` class is a data structure that contains all data loaded and processed by the application.
+The `CoilSolution` class is a data structure that contains all data loaded and processed by `pyCoilGen`. You can use this information to [...].
```
Comment (jayniep) [inline docs/source/results.md:32]: Describe why this is useful.

File: docs/source/results.md [hunk 7]
```diff
@@ -44,7 +43,7 @@ class CoilSolution:
 
 ## SolutionErrors
 
-Unless suppressed by `skip_postprocessing`, the `SolutionErrors` class contains information about the solution.
+Unless `skip_postprocessing` is set to True, the `SolutionErrors` class contains information about the solution. You can use this information to [...].
```
Comment (jayniep) [inline docs/source/results.md:46]: Describe why this is useful

File: docs/source/quick_start.md (hunk 5 above)
Comment (kev-m) [reply docs/source/quick_start.md:26]: Yes. The application has only this one interface.

File: docs/source/results.md (hunk 7 above)
Comment (kev-m) [reply docs/source/results.md:46]: Addressed in commit 3780bc6.

File: docs/source/results.md (hunk 6 above)
Comment (kev-m) [reply docs/source/results.md:32]: Addressed in commit 3780bc6.

File: docs/source/quick_start.md (hunk 4 above)
Comment (kev-m) [reply docs/source/quick_start.md:21]: Addressed in commit 3780bc6.

File: docs/source/configuration.md (hunk 1 above)
Comment (kev-m) [reply docs/source/configuration.md:479]: Addressed in commit 3780bc6.

**Pull request merged**